<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>656</width>
    <height>480</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string notr="true">HDF5 PAD</string>
  </property>
  <property name="windowIcon">
   <iconset resource="hdf5pad.qrc">
    <normaloff>:/icons/app</normaloff>:/icons/app</iconset>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <layout class="QGridLayout" name="gridLayout">
      <property name="sizeConstraint">
       <enum>QLayout::SetDefaultConstraint</enum>
      </property>
      <item row="0" column="0">
       <widget class="QLineEdit" name="edtPath"/>
      </item>
      <item row="0" column="1">
       <widget class="QPushButton" name="btnGo">
        <property name="text">
         <string>Go</string>
        </property>
        <property name="icon">
         <iconset resource="hdf5pad.qrc">
          <normaloff>:/icons/res/go.svg</normaloff>:/icons/res/go.svg</iconset>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QPushButton" name="btnUp">
        <property name="text">
         <string>Up</string>
        </property>
        <property name="icon">
         <iconset resource="hdf5pad.qrc">
          <normaloff>:/icons/res/up.svg</normaloff>:/icons/res/up.svg</iconset>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QSplitter" name="splitter_2">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <widget class="QTreeWidget" name="tree">
       <property name="columnCount">
        <number>2</number>
       </property>
       <column>
        <property name="text">
         <string notr="true">1</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string notr="true">2</string>
        </property>
       </column>
      </widget>
      <widget class="QSplitter" name="splitter">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <widget class="QFrame" name="frame">
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout">
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item>
          <widget class="QLabel" name="labelData">
           <property name="text">
            <string>TextLabel</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QTableWidget" name="tableData"/>
         </item>
         <item>
          <widget class="QComboBox" name="cbxDataPages"/>
         </item>
        </layout>
       </widget>
       <widget class="QTableWidget" name="tableAttr">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
       </widget>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <property name="movable">
    <bool>false</bool>
   </property>
   <property name="floatable">
    <bool>false</bool>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionOpen"/>
   <addaction name="separator"/>
   <addaction name="actionBack"/>
   <addaction name="actionForward"/>
   <addaction name="separator"/>
   <addaction name="actionCopy"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionOpen">
   <property name="icon">
    <iconset>
     <normalon>:/icons/res/open.png</normalon>
    </iconset>
   </property>
   <property name="text">
    <string>Open</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="action">
   <property name="text">
    <string>-</string>
   </property>
  </action>
  <action name="actionClose">
   <property name="text">
    <string>Close</string>
   </property>
  </action>
  <action name="actionBack">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="hdf5pad.qrc">
     <normaloff>:/icons/res/left-arrow.svg</normaloff>:/icons/res/left-arrow.svg</iconset>
   </property>
   <property name="text">
    <string>Back</string>
   </property>
  </action>
  <action name="actionForward">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="hdf5pad.qrc">
     <normaloff>:/icons/res/right-arrow.svg</normaloff>:/icons/res/right-arrow.svg</iconset>
   </property>
   <property name="text">
    <string>Forward</string>
   </property>
  </action>
  <action name="actionCopy">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="hdf5pad.qrc">
     <normaloff>:/icons/res/copy.svg</normaloff>:/icons/res/copy.svg</iconset>
   </property>
   <property name="text">
    <string>Copy</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <tabstops>
  <tabstop>edtPath</tabstop>
  <tabstop>btnGo</tabstop>
  <tabstop>tree</tabstop>
 </tabstops>
 <resources>
  <include location="hdf5pad.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>edtPath</sender>
   <signal>returnPressed()</signal>
   <receiver>btnGo</receiver>
   <slot>click()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>246</x>
     <y>55</y>
    </hint>
    <hint type="destinationlabel">
     <x>527</x>
     <y>55</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
